### ISSUE TITLE
Could not use databricks_test on windows, on version 0.0.3

### ISSUE THREAD

**Issue by anandmrya** [algattik/databricks_test#2]
On windows when running pytest following error occured
 
```
pyspark.sql.utils.IllegalArgumentException: 'java.net.URISyntaxException: Relative path in absolute URI: file:C:/Users/ANAND~1.MAU/AppData/Local/Temp/tmp_5luqhb7'
```
After checking online i found that when creating spark session, hive directory should be initialised like:
```
file:///C:/Users/ANAND~1.MAU/AppData/Local/Temp/tmp_5luqhb7'
```
As you can see after file: "///" are required on windows.
I checked on linux your code is working fine on that.

**Comment by algattik** [comment]
I've released 0.0.4 with this fix https://github.com/algattik/DataOps/commit/afbb5dbfadb5140dc232c5ecaa875cf7a79ae0c6, could you test it?

**Comment by anandmrya** [comment]
Running Okay, Thanks
